test/README.md

```diff
@@ -406,31 +456,37 @@ are not saved to the default variables file by the framework.
 
 ## Troubleshooting
 
-Use `./bin/scenario.sh login` to login to a VM for a scenario as the
+### Accessing VMs
+
+Use `./test/bin/scenario.sh login` to login to a VM for a scenario as the
 `redhat` user.
 
-## Cleaning up
+### Cleaning Up
+
+On a `development host`, use `./test/bin/composer_cleanup.sh` to stop any
+running jobs, remove everything from the queue, and delete existing builds.
 
-Use `./bin/composer_cleanup.sh` to stop any running jobs, remove
-everything from the queue, and delete existing builds.
+> It is recommended to use `./scripts/image-builder/cleanup.sh -full` to
+> fully clean Image Builder jobs and cache, also restarting its services.
 
-Use `./bin/scenario.sh cleanup` to remove the test infrastructure for a
-scenario.
+On a `hypervisor host`, use `./test/bin/scenario.sh cleanup` to remove the
+test infrastructure for a scenario.
 
 ```
-$ ./bin/scenario.sh cleanup  ./scenarios/rhel-9.2-microshift-source-standard-suite/
+$ ./test/bin/scenario.sh cleanup \
+      ./test/scenarios/rhel-9.2-microshift-source-standard-suite.sh
 ```
 
-## CI Integration Scripts
+### CI Integration Scripts
```

Comment:
I think this makes this section part of "Troubleshooting" but it wasn't intended to be. Maybe it should have been?